### PR TITLE
Set a Table and Tidy Up reworked

### PR DESCRIPTION
**Pull request by kyordhel**
- Addressed #252
- Removed long test in favor of a boring ans structured schema
- Test structure was preserved
- Added images on how table should look like.
- Several issues to be solved before merging
  - [ ] Test requires cleaning before and after setting the table
    - This is redundant
    - Also a waste of time
    - Too complex for the first time. Instead robot should just set the table and clean afterwards.
  - [ ] It is not specified what is the (default) table setting
  - [ ] When the option of serving a meal is used, table setting must depend on it. E.g.
    - Bowl and spoon for cereal
    - Dish, fork, and glass for toasts and juice
    - Knife, spoon, fork, cup, dish, and glass for English Breakfast
    - Dish and mug for croissant and coffee
    - Napkin and carpet always present
  - [ ] After the meal, the robot should take dirty utensils to the sink or dishwasher
    - In a first run, all to the sink (OC reorganizes)
    - Later on, robot should take dirty appliances to the sink and clean ones back to the cupboard.

### CODE REVIEW

File: tests/SetATableTidyUp.tex [hunk 1]
```diff
@@ -1,57 +1,74 @@
...
+\begin{itemize}
+	\item[Part I:] Setting up the table
+	\begin{enumerate}
+		\item \textbf{Fetching command:} The operator requests the robot to set up the table. Optionally, the robot may ask the operator \textit{what has to be served}. The operator replies with the randomly selected (and undisclosed) option.
 
-In phase~II, the operator requests the robot to clean up the table. This includes both returning the placed objects to their original location, and cleaning up dirt and spills using a cleaning cloth. This requires the robot to memorize the location the objects were originally recognized.
+		% I find this unnecessary, the table must be clear already or this test will take forever.
+		\item \textbf{Clearing the table:} The robot must analyze the table and remove all unnecessary objects.
```
Comment (LoyVanBeek) [inline tests/SetATableTidyUp.tex]: This seems redundant with Phase II and was not in the original description. I think we can just leave this bit out

**Comment by LoyVanBeek** [comment]
As for the meal options and what they entail for the table setting, this depends on the objects available and can thus only be specified at the event. We can provide some examples though

**Comment by LoyVanBeek** [comment]
Further, having a robot separate out clean and dirty dishes is something I'd like in my home but is maybe something for next year. We can make it optional now, but nobody ever does optionals.

File: tests/SetATableTidyUp.tex (hunk 1 above)
Comment (kyordhel) [inline tests/SetATableTidyUp.tex]: Agreed, I'm fixing that one right now.

**Comment by kyordhel** [comment]
### Default table setting
I think we can get *cereals* and sets of *fork, spoon, and knife* all around the world without any problem. If the option is cereal, a bowl and spoon is enough. Appliances are easy to remove but hard to add.

### Separating clean and dirty
No way for this year, that's for the years to come, but utensils should be taken to the sink afterwards assuming all are dirty after use. No fork has ever complained for being washed when clean.

**Pull request merged**